### PR TITLE
chore(bigquery): release 1.23.0

### PR DESCRIPTION
**Pull request by tswast**
This pull request was generated using releasetool.

### CODE REVIEW

File: bigquery/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,26 @@
...
+- add description for routine entities ([#9785](https://github.com/googleapis/google-cloud-python/pull/9785))
+
+### Documentation
+
+- update code samples to use strings for table and dataset IDs ([#9495](https://github.com/googleapis/google-cloud-python/pull/9495))
+
+### Internal / Testing Changes
+
+- run unit tests with Python 3.8 ([#9880](https://github.com/googleapis/google-cloud-python/pull/9880))
+- import `Mapping` from `collections.abc` not from `collections` ([#9826](https://github.com/googleapis/google-cloud-python/pull/9826))
```
Comment (plamut) [inline bigquery/CHANGELOG.md]: As per style guide, the sentences should be capitalized and concluded with a period at the end, right?

**Comment by plamut** [comment]
Note: Some of the unit tests might fail, because the transport now includes an explicit `timeout` argument (set to `None` by default), which breaks a few assertions about the calls made.

File: bigquery/CHANGELOG.md (hunk 1 above)
Comment (tswast) [reply bigquery/CHANGELOG.md]: Done in f499e7eb34395a9acf2b2e7cff5e12e2515d7077.

**Comment by tswast** [comment]
Fixed unit test failures in c10002e49475fd91dd260b05d66b7d42f8c7add9.

**Pull request merged**